rustfmt-core/rustfmt-bin/src/bin/main.rs

```diff
@@ -92,7 +92,6 @@ struct Opt {
     verbose: bool,
 
     // Nightly-only options.
-
```

Comment:
Might be nice to keep this (and the `Positional args` on line 125/126 below) separate so they don't seem to be pinned to a specific opt?

Reply:
Now rustfmt removes the newline in this case.
So if I insert newline here, self_tests is failed...
I think this is the rustfmt bug.
maybe related: #4012